packages/subgraph/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UPCOMING]
 
+### Changed
+
+- Removed `createERC1155TokenCalls`, `createTokenCalls`, `createWrappedTokenCalls` and `createDummyActions` and use the equivalent functions from `@aragon/sdk-commons-subgraph`
```

Comment:
Make sure this is up-to-date after all the changes that have happened.

For example: `createTokenCalls` was renamed by now